src/js/cover.js

```diff
@@ -1,16 +1,47 @@
 (function(){
   var coverTitle = document.getElementsByClassName('homepage-hero__title');
   var coverWrapper = document.getElementsByClassName('homepage-hero_wrapper');
-  var video = document.getElementById('cover-video');
-
-  video.oncanplay= function () {
-    setTimeout(function(){
-      coverTitle[0].classList.add('title-translate');
-      coverWrapper[0].classList.add('wrapper-translate');
-      setTimeout(function(){
-        coverTitle[0].classList.remove('title-translate');
-        coverWrapper[0].classList.remove('wrapper-translate');
-      }, 21000);
-    }, 7500);
-  };
-})();
+  var video = document.getElementById('video');
+  var play = document.getElementById('play-btn');
+  var playCover = document.getElementById('video-cover');
+    if(screen.width > 767){
+      video.oncanplay = function () {
+        setTimeout(function(){
+          coverTitle[0].classList.add('title-translate');
+          coverWrapper[0].classList.add('wrapper-translate');
+          setTimeout(function(){
+            coverTitle[0].classList.remove('title-translate');
+            coverWrapper[0].classList.remove('wrapper-translate');
+          }, 21000);
+        }, 7500);
+      };
+    }else if(screen.width < 767){
```

Comment:
identation, in webStorm select all and press Ctrl + Alt + L

Reply:
ooohh O: